node/logger/logger.go

```diff
@@ -207,3 +208,53 @@ func Panicf(format string, args ...interface{}) {
 func GetLevel() string {
 	return getInstance().GetLevel().String()
 }
+
+func getStringFromCall(request interface{}, methodName string) string {
```

Comment:
can you please add in the description log diff?

Reply:
[done](https://github.com/IBM/ibm-block-csi-driver/pull/485#issue-1185117618)